webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,17 @@ function BugForm() {
     }, this));
   };
 
+  // See function autoExpand in issues.js
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand",function(){})
+    .on("input.js-autoExpand", "textarea.js-autoExpand", function() {
+      this.style.overflow = 'hidden';
```

Comment:
Also, since we're using jQuery, you can just use the $.fn.css API for consistency. That is `$(this).css({'height': foo, 'overflow': bar'});`

See http://api.jquery.com/css/ for more docs.